app-config.heroku.yaml

```diff
@@ -47,3 +47,11 @@ kubernetes:
           authProvider: 'serviceAccount'
           caData: ${K8S_CA_DATA}
           serviceAccountToken: ${KUBERNETES_SERVICE_ACCOUNT_TOKEN}
+
+catalog:
+  providers:
+    backstage:
+      enabled: false
+      schedule:
+        frequency: { minutes: 360 } # runs every 6 hours
+        timeout: { minutes: 5 }
```

Comment:
is five minutes (especially on heroku limited runners) enough to read, unpack and inspect the whole repos, and commit the results to the catalog db? let's keep an eye on it and adjust if necessary

Reply:
Yes, changed this to 30, that should do. 👍